### PR TITLE
Add builtin test list

### PR DESCRIPTION
**Pull request by thiagomacieira**
This replaces the previous weighted test list, which was missing, with a simple sequential order.

### CODE REVIEW

**Review by thac0 — DISMISSED** [review]
👍 Solid.

File: framework/scripts/generate-test-list.pl [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/perl
+use strict;
```
Comment (thiagomacieira) [inline framework/scripts/generate-test-list.pl:2]: Missing copyright notice.

**Review by thac0 — APPROVED** [review]
👍

File: framework/scripts/generate-test-list.pl (hunk 1 above)
Comment (thac0) [reply framework/scripts/generate-test-list.pl:5]: Gah, I missed that.

**Pull request merged**